templates/apollo-starter-ts/tslint.json

```diff
@@ -14,7 +14,8 @@
       "object-literal-sort-keys": false,
       "trailing-comma": false,
       "object-literal-shorthand": false,
-      "no-console": false
+      "no-console": false,
+      "interface-name": false
```

Comment:
Reason?

Reply:
Demanding resolver name to start with "I"